src/api/articles.js

```diff
@@ -1,12 +1,13 @@
 import axios from "axios";
 
-function getArticles() {
-  return axios
-    .get("https://hollycr-nc-news.onrender.com/api/articles")
-    .then(({ data }) => {
-      const { articles } = data;
-      return articles;
-    });
+function getArticles(topic) {
+  let queryStr = "https://hollycr-nc-news.onrender.com/api/articles";
+  if (topic) queryStr += `?topic=${topic}`;
```

Comment:
I think you could do this with axios params. Take a look at the docs for it and if you wanna refactor later then go for it 🙂